### ISSUE TITLE
Motion sensors (SML001 and SML002) not working correct after update 2.0.26 and Homey 5 

### ISSUE THREAD

**Issue by RonaldNeed** [sebbebebbe/com.philips.hue.zigbee#301]
Hello Johan,
After an automatic Homey update to 5.0.0 and an update Philips Hue Zigbee to 2.0.26 this morning (20 feb 4:48) the Philips motion sensors (indoor SML001 and outdoor SML002) are not working correct anymore.
No motion is detected, no red LED on the device when motion is detected. Information on temperature, battery and light is transmitted to Homey and available on the homey app and website. Motion detected information is stopped after the update.  Rebooted the app and Homey and power cycled some motion sensors but did not solve the problem. 
Hope this is enough information to solve. 
Regards, Ronald

**Comment by JohanBendz** [comment]
Hi @RonaldNeed, I think I have found the problem and are working on a fix. Can you please interview your devices and send me the data so that I can verify my theory?  https://developer.athom.com/tools/zigbee

**Comment by JohanBendz** [comment]
Hi again. @RonaldNeed If you are able to test the app from github using CLI I have a potential fix in the latest version. https://github.com/JohanBendz/com.philips.hue.zigbee/tree/sdk3

**Comment by RonaldNeed** [comment]
Hi Johan, that a quick reply. Thxs.  I will send the "Interview" and will try to test the app.

**Comment by RonaldNeed** [comment]
Hi Johan,

Here are the interviews: SML001 following SML002.
(What about testing the app  from Github using CLI is next to findout.

Regards Ronald

```
  "ids": {
    "modelId": "SML001",
    "manufacturerName": "Philips"
  },
  "endpoints": {
    "endpointDescriptors": [
      {
        "endpointId": 2,
        "applicationProfileId": 260,
        "applicationDeviceId": 263,
        "applicationDeviceVersion": 0,
        "_reserved1": 0,
        "inputClusters": [
          0,
          1,
          3,
          1030,
          1024,
          1026
        ],
        "outputClusters": [
          25
        ]
      },
      {
        "endpointId": 1,
        "applicationProfileId": 49246,
        "applicationDeviceId": 2128,
        "applicationDeviceVersion": 0,
        "_reserved1": 2,
        "inputClusters": [
          0
        ],
        "outputClusters": [
          0,
          3,
          4,
          6,
          8,
          768,
          5
        ]
      }
    ],
    "endpoints": {
      "1": {
        "clusters": {
          "basic": {
            "attributes": [
              {
                "id": 0,
                "name": "zclVersion",
                "value": 1
              },
              {
                "id": 1,
                "name": "appVersion",
                "value": 2
              },
              {
                "id": 2,
                "name": "stackVersion",
                "value": 1
              },
              {
                "id": 3,
                "name": "hwVersion",
                "value": 1
              },
              {
                "id": 4,
                "name": "manufacturerName",
                "value": "Philips"
              },
              {
                "id": 5,
                "name": "modelId",
                "value": "SML001"
              },
              {
                "id": 6,
                "name": "dateCode",
                "value": "20160630"
              },
              {
                "id": 7,
                "name": "powerSource",
                "value": "battery"
              },
              {
                "id": 16384,
                "name": "swBuildId",
                "value": "6.1.0.18912"
              }
            ],
            "commandsGenerated": "UNSUP_GENERAL_COMMAND",
            "commandsReceived": "UNSUP_GENERAL_COMMAND"
          }
        },
        "bindings": {
          "basic": {
            "attributes": [
              {
                "id": 0,
                "name": "zclVersion",
                "value": 1
              },
              {
                "id": 1,
                "name": "appVersion",
                "value": 2
              },
              {
                "id": 2,
                "name": "stackVersion",
                "value": 1
              },
              {
                "id": 3,
                "name": "hwVersion",
                "value": 1
              },
              {
                "id": 4,
                "name": "manufacturerName",
                "value": "Philips"
              },
              {
                "id": 5,
                "name": "modelId",
                "value": "SML001"
              },
              {
                "id": 6,
                "name": "dateCode",
                "value": "20160630"
              },
              {
                "id": 7,
                "name": "powerSource",
                "value": "battery"
              },
              {
                "id": 16384,
                "name": "swBuildId",
                "value": "6.1.0.18912"
              }
            ],
            "commandsGenerated": "UNSUP_GENERAL_COMMAND",
            "commandsReceived": "UNSUP_GENERAL_COMMAND"
          },
          "identify": {
            "attributes": [],
            "commandsGenerated": "UNSUP_GENERAL_COMMAND",
            "commandsReceived": "UNSUP_GENERAL_COMMAND"
          },
          "groups": {
            "attributes": [],
            "commandsGenerated": "UNSUP_GENERAL_COMMAND",
            "commandsReceived": "UNSUP_GENERAL_COMMAND"
          },
          "onOff": {
            "attributes": [],
            "commandsGenerated": "UNSUP_GENERAL_COMMAND",
            "commandsReceived": "UNSUP_GENERAL_COMMAND"
          },
          "levelControl": {
            "attributes": [],
            "commandsGenerated": "UNSUP_GENERAL_COMMAND",
            "commandsReceived": "UNSUP_GENERAL_COMMAND"
          },
          "colorControl": {
            "attributes": [],
            "commandsGenerated": "UNSUP_GENERAL_COMMAND",
            "commandsReceived": "UNSUP_GENERAL_COMMAND"
          },
          "scenes": {
            "attributes": [],
            "commandsGenerated": "UNSUP_GENERAL_COMMAND",
            "commandsReceived": "UNSUP_GENERAL_COMMAND"
          }
        }
      },
      "2": {
        "clusters": {
          "basic": {
            "attributes": [
              {
                "id": 0,
                "name": "zclVersion",
                "value": 1
              },
              {
                "id": 1,
                "name": "appVersion",
                "value": 2
              },
              {
                "id": 2,
                "name": "stackVersion",
                "value": 1
              },
              {
                "id": 3,
                "name": "hwVersion",
                "value": 1
              },
              {
                "id": 4,
                "name": "manufacturerName",
                "value": "Philips"
              },
              {
                "id": 5,
                "name": "modelId",
                "value": "SML001"
              },
              {
                "id": 6,
                "name": "dateCode",
                "value": "20160630"
              },
              {
                "id": 7,
                "name": "powerSource",
                "value": "battery"
              },
              {
                "id": 16384,
                "name": "swBuildId",
                "value": "6.1.0.18912"
              }
            ],
            "commandsGenerated": "UNSUP_GENERAL_COMMAND",
            "commandsReceived": "UNSUP_GENERAL_COMMAND"
          },
          "powerConfiguration": {
            "attributes": [
              {
                "id": 32,
                "name": "batteryVoltage",
                "value": 26,
                "reportingConfiguration": {
                  "status": "UNREPORTABLE_ATTRIBUTE",
                  "direction": "reported"
                }
              },
              {
                "id": 33,
                "name": "batteryPercentageRemaining",
                "value": 109,
                "reportingConfiguration": {
                  "direction": "reported",
                  "attributeDataType": 32,
                  "minInterval": 300,
                  "maxInterval": 60000,
                  "minChange": 1,
                  "status": "SUCCESS"
                }
              }
            ],
            "commandsGenerated": "UNSUP_GENERAL_COMMAND",
            "commandsReceived": "UNSUP_GENERAL_COMMAND"
          },
          "identify": {
            "attributes": [
              {
                "id": 0,
                "reportingConfiguration": {
                  "status": "UNREPORTABLE_ATTRIBUTE",
                  "direction": "reported"
                }
              }
            ],
            "commandsGenerated": "UNSUP_GENERAL_COMMAND",
            "commandsReceived": "UNSUP_GENERAL_COMMAND"
          },
          "occupancySensing": {
            "attributes": [
              {
                "id": 0,
                "name": "occupancy",
                "value": {
                  "type": "Buffer",
                  "data": [
                    1
                  ]
                }
              },
              {
                "id": 1,
                "name": "occupancySensorType",
                "value": "pir"
              },
              {
                "id": 16,
                "name": "pirOccupiedToUnoccupiedDelay",
                "value": 0
              }
            ],
            "commandsGenerated": "UNSUP_GENERAL_COMMAND",
            "commandsReceived": "UNSUP_GENERAL_COMMAND"
          },
          "illuminanceMeasurement": {
            "attributes": [
              {
                "id": 0,
                "name": "measuredValue",
                "value": 4895
              },
              {
                "id": 1,
                "name": "minMeasuredValue",
                "value": 1
              },
              {
                "id": 2,
                "name": "maxMeasuredValue",
                "value": 65534
              }
            ],
            "commandsGenerated": "UNSUP_GENERAL_COMMAND",
            "commandsReceived": "UNSUP_GENERAL_COMMAND"
          },
          "temperatureMeasurement": {
            "attributes": [
              {
                "id": 0,
                "name": "measuredValue",
                "value": 1883
              },
              {
                "id": 1,
                "name": "minMeasuredValue",
                "value": -27315
              },
              {
                "id": 2,
                "name": "maxMeasuredValue",
                "value": 32767
              }
            ],
            "commandsGenerated": "UNSUP_GENERAL_COMMAND",
            "commandsReceived": "UNSUP_GENERAL_COMMAND"
          }
        },
        "bindings": {
          "ota": {
            "attributes": [],
            "commandsGenerated": "UNSUP_GENERAL_COMMAND",
            "commandsReceived": "UNSUP_GENERAL_COMMAND"
          }
        }
      }
    }
  }
```

SML002


```
  "ids": {
    "modelId": "SML002",
    "manufacturerName": "Philips"
  },
  "endpoints": {
    "endpointDescriptors": [
      {
        "endpointId": 2,
        "applicationProfileId": 260,
        "applicationDeviceId": 263,
        "applicationDeviceVersion": 0,
        "_reserved1": 0,
        "inputClusters": [
          0,
          1,
          3,
          1030,
          1024,
          1026
        ],
        "outputClusters": [
          25
        ]
      },
      {
        "endpointId": 1,
        "applicationProfileId": 49246,
        "applicationDeviceId": 2128,
        "applicationDeviceVersion": 0,
        "_reserved1": 2,
        "inputClusters": [
          0
        ],
        "outputClusters": [
          0,
          3,
          4,
          6,
          8,
          768,
          5
        ]
      }
    ],
    "endpoints": {
      "1": {
        "clusters": {
          "basic": {
            "attributes": [
              {
                "id": 0,
                "name": "zclVersion",
                "value": 1
              },
              {
                "id": 1,
                "name": "appVersion",
                "value": 2
              },
              {
                "id": 2,
                "name": "stackVersion",
                "value": 1
              },
              {
                "id": 3,
                "name": "hwVersion",
                "value": 1
              },
              {
                "id": 4,
                "name": "manufacturerName",
                "value": "Philips"
              },
              {
                "id": 5,
                "name": "modelId",
                "value": "SML002"
              },
              {
                "id": 6,
                "name": "dateCode",
                "value": "20180828"
              },
              {
                "id": 7,
                "name": "powerSource",
                "value": "battery"
              },
              {
                "id": 16384,
                "name": "swBuildId",
                "value": "6.1.0.25261"
              }
            ],
            "commandsGenerated": "UNSUP_GENERAL_COMMAND",
            "commandsReceived": "UNSUP_GENERAL_COMMAND"
          }
        },
        "bindings": {
          "basic": {
            "attributes": [
              {
                "id": 0,
                "name": "zclVersion",
                "value": 1
              },
              {
                "id": 1,
                "name": "appVersion",
                "value": 2
              },
              {
                "id": 2,
                "name": "stackVersion",
                "value": 1
              },
              {
                "id": 3,
                "name": "hwVersion",
                "value": 1
              },
              {
                "id": 4,
                "name": "manufacturerName",
                "value": "Philips"
              },
              {
                "id": 5,
                "name": "modelId",
                "value": "SML002"
              },
              {
                "id": 6,
                "name": "dateCode",
                "value": "20180828"
              },
              {
                "id": 7,
                "name": "powerSource",
                "value": "battery"
              },
              {
                "id": 16384,
                "name": "swBuildId",
                "value": "6.1.0.25261"
              }
            ],
            "commandsGenerated": "UNSUP_GENERAL_COMMAND",
            "commandsReceived": "UNSUP_GENERAL_COMMAND"
          },
          "identify": {
            "attributes": [],
            "commandsGenerated": "UNSUP_GENERAL_COMMAND",
            "commandsReceived": "UNSUP_GENERAL_COMMAND"
          },
          "groups": {
            "attributes": [],
            "commandsGenerated": "UNSUP_GENERAL_COMMAND",
            "commandsReceived": "UNSUP_GENERAL_COMMAND"
          },
          "onOff": {
            "attributes": [],
            "commandsGenerated": "UNSUP_GENERAL_COMMAND",
            "commandsReceived": "UNSUP_GENERAL_COMMAND"
          },
          "levelControl": {
            "attributes": [],
            "commandsGenerated": "UNSUP_GENERAL_COMMAND",
            "commandsReceived": "UNSUP_GENERAL_COMMAND"
          },
          "colorControl": {
            "attributes": [],
            "commandsGenerated": "UNSUP_GENERAL_COMMAND",
            "commandsReceived": "UNSUP_GENERAL_COMMAND"
          },
          "scenes": {
            "attributes": [],
            "commandsGenerated": "UNSUP_GENERAL_COMMAND",
            "commandsReceived": "UNSUP_GENERAL_COMMAND"
          }
        }
      },
      "2": {
        "clusters": {
          "basic": {
            "attributes": [
              {
                "id": 0,
                "name": "zclVersion",
                "value": 1
              },
              {
                "id": 1,
                "name": "appVersion",
                "value": 2
              },
              {
                "id": 2,
                "name": "stackVersion",
                "value": 1
              },
              {
                "id": 3,
                "name": "hwVersion",
                "value": 1
              },
              {
                "id": 4,
                "name": "manufacturerName",
                "value": "Philips"
              },
              {
                "id": 5,
                "name": "modelId",
                "value": "SML002"
              },
              {
                "id": 6,
                "name": "dateCode",
                "value": "20180828"
              },
              {
                "id": 7,
                "name": "powerSource",
                "value": "battery"
              },
              {
                "id": 16384,
                "name": "swBuildId",
                "value": "6.1.0.25261"
              }
            ],
            "commandsGenerated": "UNSUP_GENERAL_COMMAND",
            "commandsReceived": "UNSUP_GENERAL_COMMAND"
          },
          "powerConfiguration": {
            "attributes": [
              {
                "id": 32,
                "name": "batteryVoltage",
                "value": 26,
                "reportingConfiguration": {
                  "status": "UNREPORTABLE_ATTRIBUTE",
                  "direction": "reported"
                }
              },
              {
                "id": 33,
                "name": "batteryPercentageRemaining",
                "value": 143,
                "reportingConfiguration": {
                  "direction": "reported",
                  "attributeDataType": 32,
                  "minInterval": 300,
                  "maxInterval": 60000,
                  "minChange": 1,
                  "status": "SUCCESS"
                }
              }
            ],
            "commandsGenerated": "UNSUP_GENERAL_COMMAND",
            "commandsReceived": "UNSUP_GENERAL_COMMAND"
          },
          "identify": {
            "attributes": [
              {
                "id": 0,
                "reportingConfiguration": {
                  "status": "UNREPORTABLE_ATTRIBUTE",
                  "direction": "reported"
                }
              }
            ],
            "commandsGenerated": "UNSUP_GENERAL_COMMAND",
            "commandsReceived": "UNSUP_GENERAL_COMMAND"
          },
          "occupancySensing": {
            "attributes": [
              {
                "id": 0,
                "name": "occupancy",
                "value": {
                  "type": "Buffer",
                  "data": [
                    0
                  ]
                }
              },
              {
                "id": 1,
                "name": "occupancySensorType",
                "value": "pir"
              },
              {
                "id": 16,
                "name": "pirOccupiedToUnoccupiedDelay",
                "value": 0
              }
            ],
            "commandsGenerated": "UNSUP_GENERAL_COMMAND",
            "commandsReceived": "UNSUP_GENERAL_COMMAND"
          },
          "illuminanceMeasurement": {
            "attributes": [
              {
                "id": 0,
                "name": "measuredValue",
                "value": 18620
              },
              {
                "id": 1,
                "name": "minMeasuredValue",
                "value": 1
              },
              {
                "id": 2,
                "name": "maxMeasuredValue",
                "value": 65534
              }
            ],
            "commandsGenerated": "UNSUP_GENERAL_COMMAND",
            "commandsReceived": "UNSUP_GENERAL_COMMAND"
          },
          "temperatureMeasurement": {
            "attributes": [
              {
                "id": 0,
                "name": "measuredValue",
                "value": 1574
              },
              {
                "id": 1,
                "name": "minMeasuredValue",
                "value": -27315
              },
              {
                "id": 2,
                "name": "maxMeasuredValue",
                "value": 32767
              }
            ],
            "commandsGenerated": "UNSUP_GENERAL_COMMAND",
            "commandsReceived": "UNSUP_GENERAL_COMMAND"
          }
        },
        "bindings": {
          "ota": {
            "attributes": [],
            "commandsGenerated": "UNSUP_GENERAL_COMMAND",
            "commandsReceived": "UNSUP_GENERAL_COMMAND"
          }
        }
      }
    }
  }
```

**Comment by JohanBendz** [comment]
Thanks, did you try the sensor after interviewing it? (before testing the CLI code?)
Some ppl report it starts working when they did and just wanna check with you..

**Comment by RonaldNeed** [comment]
one sensor was with a exclamation mark on the app interface. After interviewing the communication was started with Homey (temp, light and batt) but no motion detecting. All others communicate but also no motion. I think I cannot do CLI to test the app. At this moment not enough knowledge. If you can make one for testing, I will test. Otherwise, a will wait for your update. No problem.

**Comment by ei0haro** [comment]
Hi!
I’m not sure I have exactly the same problem but Ill give it a shot. My outdoor sensor does not trigger flows since the V5 update. I can click the device in homey and see that motion has been detected, but flows are not triggered. 
I did install the sdk3 branch above through the cli but get the same behaviour.

**Comment by ei0haro** [comment]
Removing sensor and add again solved the issue for me

**Comment by RonaldNeed** [comment]
@ei0haro,
Thanks for the info. i will try with one of the sensors with almost no flows.  I will inform you on the results. Q: are you running with the latest SDK3 version? or the one that is released after the update on Homey 5?

**Comment by ei0haro** [comment]
> @ei0haro,
> Thanks for the info. i will try with one of the sensors with almost no flows. I will inform you on the results. Q: are you running with the latest SDK3 version? or the one that is released after the update on Homey 5?

Hi!
Before re-adding the sensor I had reverted back to the official release.

**Comment by RonaldNeed** [comment]
@ei0haro and @JohanBendz,  
Hi here is an update on my mission on the above problem.
Removed and added 3 motion sensors SML001 (indoor) for test. It looks like that they are working. 
In the UI of the APP there is a blinking alarm if motion is detected. Also the ‘log’ shows the alarm messages in time sequence. created some test flows on motion detected and output text to the timeline. This is working. So removing and adding the motion sensors solves the problem.  
Q: Can you confirm that the ‘red led’ stays off all time? Normally (before the update) the ‘red led’ turns on if motion is detected? 
Just a note: At this moment I did nothing with the outdoor sensors SML002. Only interviewing the outdoor motion sensors. The ‘log’ still shows the last date and time of motion detection before the update. Now 12 days ago. The motion alarm gives ‘looks like’ random detected times on al of the outdoor motion sensors.      
Next step is – tomorrow - to remove and add the outdoor sensors. That’s the larger job because on the 'not so easy getting to the sensors'. Hided in the gardens.

**Comment by RonaldNeed** [comment]
@JohanBendz, @ei0haro,
Hi here is update nr 2 on my mission on the above problem.
Removed and added 2 outdoor motion sensors SML002 (outdoor) for the test. 
This was not so successful.

Removed them from within the app, replaced batteries, added sensor with the procedure Add device.  Because of replaced batteries the adding proceeding started but failed every time. 

With the reset button (>10sec.) the sensor was added to the app. 

All the results were the same with all the outdoor sensors. [Batt], [Light sensor] and [Temp.] are working. Motion sensing, Log messages and LED are not working. Restarting the app in between did not help.
Next step is – tomorrow - to add (reconnect) the flows to the indoor motion sensors.
Hope this is useful info to you.

**Comment by jeroenski** [comment]
Same problem here the pir sensor of the outdoor sensor is not working after the update.

**Comment by JohanBendz** [comment]
Those that still have problems, please try the latest test version: https://homey.app/en-gb/app/com.philips.hue.zigbee/Philips-Hue-ZigBee/test/

If the drivers for Outdoor/Motion Sensor does not work, please try the new Occupancy Sensor versions.

**Comment by RonaldNeed** [comment]
Hi Johan, Its working on the test version and also with the released version in the store 2.0.32. for me this issue is solved
